src/mm-grid-column/mm-grid-column.js

```diff
@@ -13,46 +13,15 @@
 			StrandTraits.Refable
 		],
 
-		properties: {
-
-			SORT_ICON_COLOR: {
-				type: String,
-				value: Colors.A2,
-			},
-			INFO_ICON_COLOR: {
-				type: String,
-				value: Colors.A4,
-			},
-
-			SORT_DEFAULT: {
-				type: Number,
-				value: 0,
-			},
-			SORT_ASCENDING: {
-				type: Number,
-				value: 1,
-			},
-			SORT_DESCENDING: {
-				type: Number,
-				value: -1,
-			},
-			SORT_EVENT: {
-				type: String,
-				value: "column-sort",
-			},
-			RESIZE_EVENT: {
-				type: String,
-				value: "column-resize",
-			},
-			RESIZE_START_EVENT: {
-				type: String,
-				value: "column-resize-start",
-			},
-			RESIZE_END_EVENT: {
-				type: String,
-				value: "column-resize-end",
-			},
+		SORT_DEFAULT: 0,
+		SORT_ASCENDING: 1,
+		SORT_DESCENDING: -1,
+		SORT_EVENT: "column-sort",
+		RESIZE_EVENT: "column-resize",
+		RESIZE_START_EVENT: "column-resize-start",
+		RESIZE_END_EVENT: "column-resize-end",
```

Comment:
Are we loosening up on the properties block enforcement (perhaps only for all-capped pseudo-constants)?